eng/Dependencies.props

```diff
@@ -64,8 +64,8 @@ and are generated based on the last package release.
     <LatestPackageReference Include="Microsoft.DotNet.GenAPI" Version="$(MicrosoftDotNetGenApiPackageVersion)" />
     <LatestPackageReference Include="Moq" Version="4.10.0" />
     <LatestPackageReference Include="Newtonsoft.Json" Version="11.0.2" />
-    <LatestPackageReference Include="Polly.Extensions.Http" Version="2.0.1" />
-    <LatestPackageReference Include="Polly" Version="6.0.1" />
+    <LatestPackageReference Include="Polly.Extensions.Http" Version="3.0.0" />
```

Comment:
I don't see this one on NuGet.org. Is it not published yet?

Reply:
It is not, we're staging this for them to update their package 😀

Reply:
Cool. Well it's "approved" from my perspective. Once the target package is on NuGet.org and this change here gets checked in, Component Governance (CG) will auto-register it.